assets/airports/rjoa.json

```diff
@@ -0,0 +1,758 @@
+{
```

Comment:
Check the indentation throughout this file, it's inconsistent. All indents should be 4 spaces